pkg/server/router/dwn.go

```diff
@@ -79,20 +79,20 @@ func (dwnr DWNRouter) PublishManifest(ctx context.Context, w http.ResponseWriter
 	req := request.ToServiceRequest()
 	publishManifestResponse, err := dwnr.service.GetManifest(req)
 
-	if err != nil || &publishManifestResponse.Manifest == nil {
+	if err != nil || publishManifestResponse.Manifest == nil {
```

Comment:
to get around this check we should define an `IsEmpty` method on Manifest. I think one may even already exist.

Reply:
Done.